tests/installer.test.ts

```diff
@@ -113,13 +119,29 @@ describe('Installer', () => {
         progressCallback,
       });
     const { events, result } = await listenWhile(installer, func);
-    const zipfile = result as string;
+    const binaryConfig = result as ElectronBinary;
+    const { path: zipfile } = binaryConfig;
 
-    expect(isDownloaded).toBe(true);
-    expect(fs.existsSync(zipfile)).toBe(true);
-    expect(installer.state(version)).toBe(downloaded);
+    if (safe) {
+      expect(isDownloaded).toBe(true);
+      expect(fs.existsSync(zipfile)).toBe(true);
+      expect(installer.state(version)).toBe(downloaded);
+    }
 
-    return { events, zipfile };
+    return { events, binaryConfig };
+  }
+
+  async function unZipBinary(dirName = '13.1.7'): Promise<string> {
+    const extractDir = path.join(tmpdir, 'downloads', dirName);
+    fs.mkdirSync(path.join(tmpdir, 'downloads', dirName), {
+      recursive: true,
+    });
+
+    await extract(fixture('electron-v13.1.7.zip'), {
```

Comment:
question: why are we hard-coding `electron-v13.1.7.zip` in the test but making the `dirName` configurable?

Reply:
If I understand if correctly, I was using that for testing and debugging. Removed it now.